rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -60,7 +60,7 @@ The activity parameters for each process are created in RPA Builder and are give
 Set a value by entering text or by clicking on a checkbox, and complete your entry by pressing ENTER.
 +
 [[test-configuration-link-globals]]
-Alternatively, you can set the value of an activity parameters by linking it with a global variable. To do this, click the *Link a global activity parameter* icon image:linkglobalvariable-icon.png[ symbol,1.5%,1.5%] and select a variable for linking. RPA Manager displays variables that are of the same type and are assigned to the test phase, and only those credentials for which you belong to the xref:processautomation-prepare.adoc#form-create-credential-useraffiliation[affiliated users]. If a credential affiliated to another user is already linked, RPA Manager displays a row of stars to hide the data. 
+Alternatively, you can set the value of an activity parameters by linking it with a global variable. To do this, click the *Link a global activity parameter* icon image:linkglobalvariable-icon.png[ symbol,1.5%,1.5%] and select a variable for linking. RPA Manager displays variables that are of the same type and are assigned to the test phase, and only those credentials for which you belong to the xref:processautomation-prepare-deployment-credential.adoc#form-create-credential-useraffiliation[affiliated users]. If a credential affiliated to another user is already linked, RPA Manager displays a row of stars to hide the data. 
```

Comment:
```suggestion
Alternatively, you can set the value of an activity parameter by linking it with a global variable. To do this, click the *Link a global activity parameter* icon image:linkglobalvariable-icon.png[symbol,1.5%,1.5%] and select a variable to link. RPA Manager displays variables of the same type, assigned to the test phase, and only those credentials for which you belong to the xref:processautomation-prepare-deployment-credential.adoc#form-create-credential-useraffiliation[affiliated users]. If a credential affiliated with another user is linked, RPA Manager displays a row of stars to hide the data. 
```

Reply:
Those edits might be incorrect. 
The third sentence is hard for me to understand. Is a credential the same as a variable?

Reply:
Is the image an icon? If so, I think give it empty alt text.

Reply:
A credential is a special type of global variable. Global variables and credentials are managed in different view. The linking process is the same, but the credential has some additional security features (only visible to / usable by affiliated users, passwords are hidden and encrypted)

Reply:
Like global variables, you link credentials to activity parameters when you create or edit a run configuration.
When you edit a run configuration that contains a credential that is not affiliated with you, you can only see asterisks and not the data. If you change the credential you only see credentials affiliated with you. (Therefore, after saving, you cannot change the link back to it's previous credential.)

Reply:
I will leave the third sentence as is.